pygpu/gpuarray.pyx

```diff
@@ -1936,7 +1938,7 @@ cdef class GpuArray:
             free(steps)
 
     def __setitem__(self, idx, v):
-        cdef GpuArray tmp = self.__cgetitem__(idx)
+        cdef GpuArray tmp = self.__getitem__(idx)
```

Comment:
Don't do that.  The point of \_\_cgetitem__ was to avoid python overhead here (which is significant).